### DISCUSSION TITLE
Potion crafting

### DISCUSSION THREAD

**Discussion by Ansuz29** [bridge-core/bridge.#319] [Q&A]
I've been trying to create a custom potion recipe.

Essentially, Im trying to brew potions of decay from wither roses and regen potions. However no matter what I change I still cant brew anything from the roses, even after using the document template
`{
"format_version": "1.16.100",
"minecraft:recipe_brewing_mix": {
"description": {
  "identifier": "bridge:brew_wither"
  },
 
 "tags": [ "brewing_stand" ],
 
  "input": "minecraft:potion_type:regeneration",
  "reagent": "minecraft:wither_rose",
  "output": "minecraft:potion_type:decay",
 
  }
}`

If anyone knows what Im doing wrong some guidance would be much appreciated!

**Comment by solvedDev** [comment]
There's a second brewing recipe type "minecraft:recipe_brewing_container". Maybe give this one a try!
